src/test/resources/Features/OCD-1970-ModificationOfG1G2Values.feature

```diff
@@ -6,34 +6,28 @@ Scenario Outline: Mapping legacy values for G1/G2
 	When I open details for criteria "<CRITERIA>"
 	Then Measure Successfully Tested for "<gOption>" should display "<Measure_Name>" for "<CRITERIA>" 
 	When I look at  Successfully Tested G1G2 Measures
-	Then the "<Measure_Name>" shows Required Text as "<Required_Text>" and associated criteria as "<CRITERIA>"
+	Then the Required Test shows as "<Required_Test>" for "<Measure_Name>" and associated criteria "<CRITERIA>"
 	Examples:
-    |DB_ID|CRITERIA|gOption|Measure_Name|Required_Text|
+    |DB_ID|CRITERIA|gOption|Measure_Name|Required_Test|
```

Comment:
Think `_Test` should be `_Text`

Reply:
The field name is 'Required Test' , so i changed it to have the field name